src/index.ts

```diff
@@ -4,6 +4,8 @@ export { createScrollbars } from './common/index';
 
 /* components */
 export * from './Anchor/Anchor';
+export * from './Divider/Divider';
+export * from './List/List';
```

Comment:
Potential leftover here.

Reply:
I'd also suggest keeping the exports where they were, otherwise we'll have conflicts across PRs.